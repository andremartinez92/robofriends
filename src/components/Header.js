// @flow

import React from 'react';

function Header() {
  return <h1>Robofriends</h1>;
}

export default Header;
