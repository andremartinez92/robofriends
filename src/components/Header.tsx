import React from 'react';

function Header(): JSX.Element {
  return <h1 className="f1">Robofriends</h1>;
}

export default Header;
