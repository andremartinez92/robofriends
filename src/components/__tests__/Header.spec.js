import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../Header';

describe('<Header>', () => {
  it('renders the expected component', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
