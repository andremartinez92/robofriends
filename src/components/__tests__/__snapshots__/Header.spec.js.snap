// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> renders the expected component 1`] = `
<h1
  className="f1"
>
  Robofriends
</h1>
`;
